### PR TITLE
Created data visualization script

### PR DESCRIPTION
**Pull request by luckyberen**
Created the eda_visualization_script.py. Tested that it can work and save figures to the results folder.

### CODE REVIEW

File: src/eda_visualization_script.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"""
+Creating a exploratory histogram of the dataset.
```
Comment (poddarswakhar) [inline src/eda_visualization_script.py:2]: @luckyberen can you change this to an! haha small minor one

**Review by poddarswakhar — CHANGES_REQUESTED** [review]
just a very very little an problem haha! good job man!

File: src/eda_visualization_script.py (hunk 1 above)
Comment (luckyberen) [reply src/eda_visualization_script.py]: Already fixed!

**Review by poddarswakhar — APPROVED** [review]
Looks fab!

**Comment by poddarswakhar** [comment]
Merging to main!

**Pull request merged**